database/postgres/configuration_test.go

```diff
@@ -93,6 +95,20 @@ func generateConfig() *datastore.Configuration {
 		UID:                ulid.Make().String(),
 		IsAnalyticsEnabled: true,
 		IsSignupEnabled:    false,
-		StoragePolicy:      &datastore.DefaultStoragePolicy,
```

Comment:
I think we can delete the default storage policy

Reply:
We still use it in CreateConfigService